src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -174,7 +177,15 @@ private void Initialize(string assemblyPath, IDictionary settings)
         /// <summary>
         /// Gets a dictionary of settings for the FrameworkController
         /// </summary>
-        public IDictionary Settings { get; private set; }
+        public IDictionary Settings
+        {
+            get { return (IDictionary)InternalSettings; }
+        }
+
+        /// <summary>
+        /// Gets a dictionary of settings for the FrameworkController
+        /// </summary>
+        internal IDictionary<string, object> InternalSettings { get; private set; }
```

Comment:
@oznetmaster I told you that third party developers were using the Framework API but that I would have to ask them how they were using it. I couldn't ask them until I had some questions answered. I asked you not to make changes based on speculation, but to first determine for me if the API was broken. That's still where we a

It sounds like "using the framework API directly" translated for you into having a direct reference to the FrameworkController. If anyone is doing that, then they are not using the Framework API, which is specifically defined as a Reflection API. I could have easily clarified that for you if you had asked rather than focusing on your criticisms of the level of documentation.